src/main/java/com/autotune/analyzer/deployment/AutotuneDeployment.java

```diff
@@ -852,4 +855,148 @@ private static void addLayerInfoToApplication(ApplicationServiceStack applicatio
 		LOGGER.info("Added layer " + autotuneConfig.getName() + " to stack " + applicationServiceStack.getStackName());
 		applicationServiceStack.getApplicationServiceStackLayers().put(autotuneConfig.getName(), autotuneConfigCopy);
 	}
+
+	/**
+	 * Get Performance Profile objects from kubernetes, and watch for any additions, modifications or deletions.
+	 *
+	 * @throws IOException
+	 */
+	public static void getPerformanceProfiles() throws IOException {
```

Comment:
This code needs to be inside its own class. There is already too much code in this class that makes it difficult to parse

Reply:
Done